src/main/java/worker/Main.java

```diff
@@ -64,12 +73,14 @@ public static void startWorker(int port) {
 
     ActorSystem system = ActorSystem.create("WorkerSystem", conf);
 
-    Set<ActorSelection> initialContacts = new HashSet<ActorSelection>();
+    Set<ActorPath> initialContacts = new HashSet<>();
     for (String contactAddress : conf.getStringList("contact-points")) {
-      initialContacts.add(system.actorSelection(contactAddress + "/user/receptionist"));
+      initialContacts.add(ActorPaths.fromString(contactAddress + "/system/receptionist"));
     }
```

Comment:
My reasoning was to keep any unnecessary complexity out of there as it is anyways so many little details to take in. But I don't feel strongly about it, so if you think it is better I'll go for a separate config block.